weave/util.py

```diff
@@ -21,25 +21,17 @@ def init_sentry():
     except ImportError:
         return
     # Disable logs going to Sentry. Its slow
-    sentry_sdk.init(integrations=[LoggingIntegration(level=None, event_level=None)])
+    sentry_sdk.init(integrations=[LoggingIntegration(level=None)])
```

Comment:
why the change here?

Reply:
Reverted, was trying to debug why Sentry events weren't showing up with my latest change but that's resolved now.